src/postprocessor.rs

```diff
@@ -270,4 +291,12 @@ mod test {
 
         assert_eq!(Ok(vec![b"c25vdA==".to_vec()]), post.process(0, 0, b"snot"));
     }
+
+    #[test]
+    fn textual_length_prefix_postp() {
```

Comment:
how would this postprocessor handle an empty `data` array? worth adding a test for it?